zcash_client_sqlite/src/wallet/common.rs

```diff
@@ -157,7 +157,8 @@ where
                     ON accounts.id = {table_prefix}_received_notes.account_id
                  INNER JOIN transactions
                     ON transactions.id_tx = {table_prefix}_received_notes.tx
-                 WHERE {table_prefix}_received_notes.account_id = :account
+                 WHERE value >= 5000 -- FIXME, allow seslection of a dust input
```

Comment:
```suggestion
                 WHERE value >= 5000 -- FIXME, allow selection of a dust input
```
Also small nit: if this were done in the first `AND` clause, it would have been a one-line change instead of a two-line change.